deployments/helm/nvidia-dra-driver-gpu/templates/_helpers.tpl

```diff
@@ -169,3 +169,18 @@ Get all namespaces (driver namespace + additional namespaces from environment va
 {{- end }}
 {{- join "," $namespaces -}}
 {{- end -}}
+
+{{/*
+Get the latest available resource.k8s.io API version
+Returns the highest available version or empty string if none found
+*/}}
+{{- define "nvidia-dra-driver-gpu.resourceApiVersion" -}}
+{{- if .Capabilities.APIVersions.Has "resource.k8s.io/v1" -}}
+resource.k8s.io/v1
+{{- else if .Capabilities.APIVersions.Has "resource.k8s.io/v1beta1" -}}
```

Comment:
@klueska question -- why are we preferring `resource.k8s.io/v1beta1` over `resource.k8s.io/v1beta2`?

Reply:
No reason, it should be reversed

Reply:
It should be fine in practice.

* In 1.32 all that was available was `v1beta1`
* In 1.33, I'm pretty sure most people either (1) only continued to enable `v1beta1`, or (2) enabled both `v1beta1` and `v1beta2` (otherwise none of our stuff would have worked since we only supported `v1beta1` without this patch). 
* In 1.34, people are enabling anything explicitly and just expecting `v1` to be there and work.